### PR TITLE
diff and refactor

### PR DESCRIPTION
**Pull request by sarbbottam**
Initially wanted to address #6, `file/dir structure` started looking a bit confusing thus the following PR.

None of the changes should affect the consumer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "travis-after-all": "travis-after-all && (npm run report-coverage & npm run semantic-release)"
   },
   "bin": {
-    "eslint-find-rules": "src/bin.js"
+    "eslint-find-rules": "src/bin/find.js",
+    "eslint-diff-rules": "src/bin/diff.js"
```
Comment (kentcdodds) [inline package.json]: Should probably be documented.

Comment (sarbbottam) [inline package.json]: Will update the `readme` first thing in the morning.

Comment (sarbbottam) [inline package.json]: updated 📖

**Comment by kentcdodds** [comment]
LGTM

**Pull request merged**